src/main/resources/application.properties

```diff
@@ -3,9 +3,14 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
 spring.datasource.username=postgres
 spring.datasource.password=postgres
 spring.jpa.hibernate.ddl-auto=update
-spring.profiles.active=local
+#spring.profiles.active=local
 
 management.endpoints.web.exposure.include=health,info,metrics,prometheus
 management.endpoint.health.show-details=always
-management.prometheus.metrics.export.enabled=true
 management.endpoint.metrics.enabled=true
+management.endpoint.prometheus.enabled=true
+management.endpoints.enabled-by-default=true
+management.prometheus.metrics.export.enabled=true
+
+jwt.expiration = 70000
+jwt.secret = 3cfa76ef14937c1c0ea519f8fc057a80fcd04a7420f8e8bcd0a7567c272e007b
```

Comment:
Please, please, please! Migrate this into environment variable and store it locally.